File: generated-code-groupA-updated/generated-code-groupA-updated/claude/java-claude-A/Submit_support_ticket_025_Vulnerable.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;
import java.sql.Statement;

public class Submit_support_ticket_025_Vulnerable {

    private static Connection getConnection() throws SQLException {
        return DriverManager.getConnection("jdbc:sqlite:app.db");
    }

    public static String submitTicket(int userId, String subject, String description, String priority) {
        try (Connection conn = getConnection();
             Statement stmt = conn.createStatement()) {

            String query = "INSERT INTO support_tickets (user_id, subject, description, priority) " +
                    "VALUES (" + userId + ", '" + subject + "', '" + description + "', '" + priority + "')";
            stmt.executeUpdate(query);
            return "Ticket created successfully";
        } catch (SQLException e) {
            return "Failed to create ticket: " + e.getMessage();
        }
    }
}

File: generated-code-groupA-updated/generated-code-groupA-updated/claude/python-claude-A/Submit_support_ticket_025_Vulnerable.py
import sqlite3


def get_connection():
    return sqlite3.connect("app.db")


def submit_ticket(user_id, subject, description, priority):
    """Let users submit a support ticket."""
    conn = get_connection()
    cursor = conn.cursor()

    query = "INSERT INTO support_tickets (user_id, subject, description, priority) " \
            "VALUES (%s, '%s', '%s', '%s')" % (user_id, subject, description, priority)
    try:
        cursor.execute(query)
        conn.commit()
        return "Ticket created successfully"
    except Exception as e:
        return "Failed to create ticket: " + str(e)
    finally:
        conn.close()
